accepted/00063-hub-iss.md

```diff
@@ -0,0 +1,246 @@
+- Feature Name: hub_iss
+- Start Date: 2019-10-17
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Uyuni Server has a functionality called Inter-Server Synchronization (abbreviated to ISS from now on) that allows users to transfer channel data (packages and metadata) from one Server (called an ISS Master) to another Server (called an ISS Slave).
+
+Current implementation [is documented in Uyuni manuals](https://www.uyuni-project.org/uyuni-docs/uyuni/administration/iss.html).
+
+With the Hub architecture, there will be multiple Servers that receive content managed in the Hub, hence ISS will be a required mechanism.
+
+This RFC describes an overhaul of the ISS implementation in order to cover more use cases (in particular, synchronization of content different than channel data) and with better performance to attain higher scalability.
+
+See the [Hub general RFC](accepted/00060-hub-general.md) for an introduction to the Hub project.
+
+# Motivation
+[motivation]: #motivation
+
+ISS is currently limited to software channels, and it is not sure whether it can scale up above tens of Servers.
+
+From the requirements:
+
+> * Managed content can be selectively synced from Hub to Servers
+>   * In order of importance:
+>    - Configuration Management data (states, pillars...)
```

Comment:
Should we list also software channels, product and subscription data here? This exists in the current ISS implementation but if you want to draw a full picture I think this is needed.

Reply:
Copypaste error, this was in the requirements.

https://github.com/uyuni-project/uyuni-rfc/blame/master/accepted/00060-hub-general.md#L34

Fixed.